src/main/java/org/folio/circulation/domain/ItemStatus.java

```diff
@@ -10,11 +10,12 @@
   AWAITING_PICKUP("Awaiting pickup"),
   AWAITING_DELIVERY("Awaiting delivery"),
   CHECKED_OUT("Checked out"),
+  IN_PROCESS("In process"),
   IN_TRANSIT("In transit"),
   MISSING("Missing"),
-  PAGED("Paged"),
   ON_ORDER("On order"),
-  IN_PROCESS("In process"),
+  PAGED("Paged"),
+  WITHDRAWN("Withdrawn"),
```

Comment:
Have these new states been included in the request white / black list? And in rules for checking out?

Reply:
No, I can remove these new statuses, they were added just to be consistent with the enum values in the schema.

Does it work?

Reply:
I think we need to find out what circulation should do with these statuses, please can you follow up with Emma and Cate.

I'm slightly surprised `Paged` wasn't present, as that is a status that circulation actively uses.

Reply:
Sorry, it seems that values are reordered. The only new status here is `WITHDRAWN`.

Reply:
What does withdrawn mean?

Reply:
I don't know, maybe it is an order-related status. But this status is present on the inventory.item search tab:
![image](https://user-images.githubusercontent.com/9094650/71823542-2280fa00-30a0-11ea-803f-e445045fd96f.png)

Reply:
I don't know, maybe it is an order-related status. But this status is present on the inventory.item search tab:
![image](https://user-images.githubusercontent.com/9094650/71823542-2280fa00-30a0-11ea-803f-e445045fd96f.png)

Reply:
Please can you follow up with Cate, Charlotte and Emma to try to identify why it was introduced and how circulation should handle that status.

Reply:
I'll move it as a separate ticket.

Reply:
Removed as per clarification.